#71 Shape Preservation Documentation (#73)
Inbibo/adonisfx_docs: docs/maya/muscle.md

#### Stiffness Settings
| Name | Type | Default | Animatable | Description |
| :--- | :--- | :------ | :--------- | :---------- |
| **Use Custom Stiffness**                  | Boolean | False          | ✓ | Toggles the use of a custom stiffness value. If custom stiffness is used, *Material* and *Stiffness Multiplier* will be disabled and *Stiffness* will be used instead. |
| **Stiffness**                             | Float   | 10<sup>5</sup> | ✓ | Sets the custom stiffness value. Its value must be greater than 0.0. |
| **Override Shape Preservation Stiffness** | Boolean | False          | ✓ | Override the shape preservation stiffness with a custom value. If disabled it will use either the material stiffness or the custom stiffness value. |

> | **Shape Preserve Stiffness**              | Float   | 1000.0 | ✓ | Sets the override for the shape preservation stiffness to control the amount of shape preservation to be applied. |

Reviewer: May you add some horizontal spaces to align the columns? This comment applies to all deformers.

Also I like more the following suggestion, the current description feels like we are describing weights.

```suggestion
| **Shape Preserve Stiffness** | Float | 1000.0 | ✓ | Sets the stiffness shape preservation override value. Its value must be greater than 0.0 |
```

Author: Done!

| **Fibers**                      | {0.0, 0.0, 0.0} | The deformer estimates the fiber directions at each vertex based on the tendon weights. In case that the estimated fibers do not fit well to the desired directions, the paint tool can be used to comb the fibers manually. The fibers can be displayed using the *Muscle Fibers* option in the [debugger](#debugger). |
| **Global Damping**              | 1.0             | Set global damping per vertex in the simulated mesh. The greater the value per vertex is the more it will attempt to retain its previous position. |
| **Masses**                      | 1.0             | Set individual mass values per vertex in the simulated mesh. |

> | **Shape Preservation**           | 0.1             | Amount of correction to apply to the current vertex to maintain the initial state of the 3 vertices conforming a single shape preservation constraint. |

Reviewer: ```suggestion
| **Shape Preservation**          | 0.1             | Amount of correction to apply to the current vertex to maintain the initial shape state with the surrounding vertices. |
```

It is not that clear for me. I like this one more but also is not perfect. This comment might be applied to all the deformers.

Author: I see your point! I have rewritten this description
